src/terminal_pane/terminal_pane.rs

```diff
@@ -584,6 +576,19 @@ impl vte::Perform for TerminalPane {
             };
             self.scroll.delete_lines_in_scroll_region(count);
             self.scroll.add_empty_lines_in_scroll_region(count);
+        } else if c == 'B' {
+            /*
+             * 002 02 B STX	Start of Text, implied end of header
+             */
+            if let Some(move_cursor_back) = params.get(1) {
+                let move_cursor_back = *move_cursor_back;
+                if move_cursor_back > 0 {
+                    self.move_cursor_backwards(move_cursor_back as usize);
+                }
+            }
+        } else if c == 'p' {
+            self.clear_scroll();
+            self.reset_size_and_position_override();
```

Comment:
I'm not sure I understand... we use `reset_size_and_position_override` when the pane was full_screen (had a size and position override) and we want to bring it back. Why do we need it in this case?

Reply:
Sorry, I saw `reset` and thought: reset!